test-app/.editorconfig

```diff
@@ -0,0 +1,19 @@
+# EditorConfig helps developers define and maintain consistent
```

Comment:
so I've left `.editorconfig` in the root as it was and copied to the test-app folder so that:

1. `.editorconfig` applies to the whole repository
2. the test app is as close to the stock Ember CLI app as we can have it.